pkg/coderefs/search.go

```diff
@@ -40,6 +40,11 @@ func (lines searchResultLines) Swap(i, j int) {
 
 // paginatedSearch uses approximations to decide the number of flags to scan for at once using maxSumFlagKeyLength as an upper bound
 func paginatedSearch(cmd command.Searcher, flags []string, maxSumFlagKeyLength, ctxLines int, delims []rune) ([][]string, error) {
+	searchType := "flags"
+	if delims == nil {
```

Comment:
We don't support searching for the flag key itself without delimeters, so this conditional is fine